lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -856,3 +856,70 @@ def test_streamplot():
     ax.streamplot(x, y, u, v, transform=ccrs.PlateCarree(),
                   density=(1.5, 2), color=mag, linewidth=2 * mag)
     return fig
+
+
+@pytest.mark.natural_earth
+@pytest.mark.parametrize(
+    'map_projection',
+    [ccrs.Mercator(), ccrs.InterruptedGoodeHomolosine(), ccrs.Robinson()]
```

Comment:
```suggestion
    [ccrs.Mercator(), ccrs.InterruptedGoodeHomolosine(), ccrs.Robinson()]
```
I guess I don't see a lot of benefit from multiple projections here and the downside is it grows the repo size, so I'd personally vote to keep it at just one of your choice here.

Reply:
fair enough.  I added homolosine originally to verify that annotations bridged the gaps and extended outside of the map areas correctly.  So, I'll leave that one in, even thought it's a little ridiculous.